microsoft-edge/web-platform/web-platform.md

```diff
@@ -25,7 +25,7 @@ These aspects of developing for the web platform are described below.
 <!-- ====================================================================== -->
 ## Site compatibility-impacting changes coming to Microsoft Edge
 
-This article lists differences between the schedule of changes for Microsoft Edge versus the Chromium project, and high-impact changes which the Microsoft Edge team is tracking especially closely.
+This article lists the schedule of changes for Microsoft Edge and the Chromium project.  It also highlights any differences and high-impact changes which the Microsoft Edge team is tracking especially closely.
```

Comment:
This change looks good to me, but the same sentence also appears on this page: `microsoft-edge/web-platform/site-impacting-changes.md`, so it should be changed there too.

Reply:
fixed

Reply:
Maybe you forgot to push a commit, but the article I mentioned still has the old sentence, not the new one you wrote here.